Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM docker.elastic.co/elasticsearch/elasticsearch:5.5.2
```

Comment:
I think we have to decide how we backport the repo for different ES versions,
For example, I think that keeping the `master` in the most recent ES version and having different branches for other versions is a good idea. 

When it's decided the dockerfile version should change 👍